src/nvim/memline.c

```diff
@@ -3856,7 +3856,7 @@ static void ml_updatechunk(buf_T *buf, linenr_T line, long len, int updtype)
  * Find offset of line if "lnum" > 0
  * return -1 if information is not available
  */
-long ml_find_line_or_offset(buf_T *buf, linenr_T lnum, long *offp)
+long ml_find_line_or_offset(buf_T *buf, linenr_T lnum, long *offp, bool no_ff)
```

Comment:
Could you add minimal documentation about `no_ff`?

Reply:
Done.